crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -253,6 +263,7 @@ pub struct Config {
     pub ethereum: Ethereum,
     pub rpc_address: SocketAddr,
     pub rpc_cors_domains: Option<AllowedOrigins>,
+    pub ws: WebSocket,
```

Comment:
I think its better to make this an `Option<WebSocket>` and remove the `enabled` field from it. Its more type safe that way, as you can only access the parameters if it actually exists.

Reply:
+1

Reply:
Done ;)